documentation/PreferencesServer.md

```diff
@@ -349,6 +349,202 @@ There are two important things to note here:
    to `http://registry.gpii.net/common/fontSize`. Since we do not allow the same setting to be present multiple times in
    the NP set, the fontSize has been stored in the flat ontology and removed from the ISO24751 block.
 
+### GET /prefssafe/:prefsSafeId

```

Comment:
For each endpoint, it would be helpful to add examples of a query URL, input and output (success or fail) payloads.

Reply:
Hi! I believe these should all be added now.